src/compiler/checker.ts

```diff
@@ -390,7 +390,7 @@ namespace ts {
         const evolvingArrayTypes: EvolvingArrayType[] = [];
         const undefinedProperties = createMap<Symbol>() as UnderscoreEscapedMap<Symbol>;
 
-        const unknownSymbol = createSymbol(SymbolFlags.Property, "unknown" as __String);
+        const unknownSymbol = createSymbol(SymbolFlags.Property, "<<unknown>>" as __String);

```

Comment:
This change doesn't seem necessary; I don't think we show this to the user much.